### DISCUSSION TITLE
Is there something like photoprism for voice messages?

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [photoprism/photoprism#3411] [General]
Heho
Do you know a tool like photoprism for voice messages?

It would be great if photoprism has such an option, but I understand that this is a complete different field. XD

Ref: https://github.com/photoprism/photoprism/discussions/780

**Comment by lastzero** [comment]
You mean for audio recordings? We could add support for audio files at some point, but it might not make much sense to "show" them as search results or in slideshows (only in combination with images)?

**Reply by DunklerPhoenix** [reply]
😲 that sounds great. In combination with images would be enough i think, but it should be possible to add multiple audio files to one image. Then I am totally happy :D

**Reply by lastzero** [reply]
That seems possible if you give us enough time to work on the other high priority feature requests first :)

**Reply by DunklerPhoenix** [reply]
Hahaha There's no hurry
